CHANGES.md

```diff
@@ -19,6 +19,8 @@
   consider a type declaration recursive if the type appeared inside an attribute
   payload (#299, @NathanReb)
 
+- Added `error_extensionf` function to the `Locate` module (#316, @panglesd)
```

Comment:
```suggestion
- Added `error_extensionf` function to the `Location` module (#316, @panglesd)
```

Also, similarly to [here](https://github.com/ocaml-ppx/ppxlib/pull/315#discussion_r797739592): would you mind rebasing over main and moving this entry up to the `unreleaed` section (it's currently in the section of the last release).